src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
-public class MyHashMap<K, V> implements MyMap<K, V> {
+import java.util.Objects;
+
+public class MyHashMap<K,V> implements MyMap<K,V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private Node<K,V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = (Node<K, V>[]) new Node[INITIAL_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        resize();
+        Node<K,V> bucket = table[getBucketIndex(hash(key))];
```

Comment:
you are not iterating over the buckets, but over the nodes
```suggestion
        Node<K,V> node = table[getBucketIndex(hash(key))];
```